packages/backend/src/routes/attestation.mjs

```diff
@@ -0,0 +1,13 @@
+import catchError from '../helpers/catchError.mjs'
+
+export default ({ app, db, synchronizer }) => {
+  const handler = async (req, res) => {
+    let attestation = await db.findOne('Attestation', {
```

Comment:
Anytime you create a variable you should use `const` instead of `let`. This lets the reader know that the variable will not (can not) be re-assigned.

The general rule is, use `let` only when you need to (e.g. when you need to re-assign).